apps/communications/contacts/views/details/js/details_controller.js

```diff
@@ -146,9 +149,13 @@
     }
   }
 
+  function setContact(contactID) {
+    _contactID = contactID;
+  }
+
   function handleEditAction(evt) {
-    // In the future the navigation will change the URL to navigate
-    // to #update view: Bug 1169579
+    window.location.href =
+      '/contacts/views/form/form.html?action=update&contact=' + _contactID;
```

Comment:
hummm ... seems we will need the url generator as an external dep, what do you think?